testhelpers/tck/README.md

```
@@ -0,0 +1,305 @@
+To use the TCK, you must implement the `TCKSuite` interface:
+
+```go
+// TCKSuite is the interface that must be implemented by the test suite.
+// It defines how to create a new store with a container.
+// The generic parameters are the storage type, the driver type returned by the Conn method,
+// and the container type used to back the storage.
+//
+// IMPORTANT: The container type must exist as a Testcontainers module.
+// Please refer to the [testcontainers] package for more information.
+type TCKSuite[T storage.Storage, D any, C testcontainers.Container] interface {
+	// NewStore is a function that returns a new store.
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)